### PR TITLE
Add translator example

### PR DESCRIPTION
**Pull request by kbrgl**
Adds a translator app to the examples. The app enables you to translate text and post the translation as a message.

### CODE REVIEW

**Comment by kbrgl** [comment]
image

**Comment by kbrgl** [comment]
https://user-images.githubusercontent.com/13417496/202175352-62c27373-ead8-4b09-a9dd-484f69ec55e6.mov

File: translator/translator/translator.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        text,
+        font_size="0.7rem",
+        font_weight="bold",
+        text_transform="uppercase",
+        letter_spacing="0.05rem",
+        **kwargs,
+    )
+
+
+def output():
```
Comment (picklelo) [inline translator/translator/translator.py:82]: This looks sick!!

File: translator/translator/translator.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        pc.select(
+            list(langs.keys()),
+            value=State.lang,
+            placeholder="Select a language",
+            on_change=State.set_lang,
+            margin_top="1rem",
+        ),
+        pc.input(
+            placeholder="Text to translate",
+            on_change=State.set_text,
```
Comment (picklelo) [inline translator/translator/translator.py:116]: We should probably make this `on_blur` for performance reasons (or add some sort of debouncing logic). If they type fast it acts strange.

File: translator/translator/translator.py [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+from datetime import datetime
+import googletrans
+
+import pynecone as pc
+from pynecone.base import Base
+
+from .langs import langs
+
+
+class Message(Base):
```
Comment (picklelo) [inline translator/translator/translator.py:13]: Maybe we include the original text, as well as the language here.

File: translator/translator/translator.py [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        border="1px solid #eaeaef",
+        margin_top="1rem",
+        border_radius="8px",
+        position="relative",
+    )
+
+
+def index():
+    """The main view."""
+    return pc.box(
```
Comment (picklelo) [inline translator/translator/translator.py:105]: Let's make this a `pc.container` to center it.

**Review by picklelo — CHANGES_REQUESTED** [review]
Looks good, just a couple comments

File: translator/translator/translator.py [hunk 5]
```diff
@@ -54,15 +58,40 @@ def header():
     )
 
 
+def down_arrow():
+    return pc.box(
```
Comment (picklelo) [inline translator/translator/translator.py:62]: We could also use a `pc.icon` here: https://pynecone.io/docs/library/media/icon

File: translator/pcconfig.py [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# The Pynecone configuration file.
```
Comment (picklelo) [inline translator/pcconfig.py:1]: We recently updated the config format - could you update this?

You'll need to
1. delete this `pcconfig.py` file
2. run `pc init`with the latest version of Pynecone (pip 0.1.5 or the latest git). It should regenerate this file and look something like this:
```
import pynecone as pc

config = pc.Config(
    app_name="translator",
)
```
3. Running `pc init` after deleting the config file will replace your app code with the template app, you should be able to do `git checkout translator` to get the code back in the main folder.

**Review by picklelo — CHANGES_REQUESTED** [review]
Thanks for the updates, just one comment about updating the config file to the latest format.

**Review by picklelo — APPROVED** [review]
LGTM!

**Pull request merged**